pkg/preference/preference.go

```diff
@@ -115,7 +115,7 @@ func NewPreferenceInfo() (*PreferenceInfo, error) {
 	configFile, err := getPreferenceFile()
 	glog.V(4).Infof("The configFile is %+v", configFile)
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to get odo config file")
+		return nil, errors.Wrap(err, "unable to get odo preference file")
```

Comment:
Shouldn't we also change https://github.com/openshift/odo/pull/1912/files#diff-5f21db40ee2e66f00f206912f4dddc2fR116 to write `preferenceFile` instead of `configFile`? Using `configFile` and `preferenceFile` interchangeably could cause ambiguity, no?

Reply:
+1, I am going to update it in the next commit.

Reply:
Done